plugins/modules/vmware_host_snmp.py

```diff
@@ -73,6 +73,18 @@
     type: str
     choices: [ debug, info, warning, error ]
     default: info
+  sys_contact:
+    description:
+        - System contact who manages the system.
+    type: str
+    version_added: '1.17.0'
+    version_added: '1.17.0'
+  sys_location:
+    description:
+        - System location.
+    type: str
+    version_added: '1.17.0'
+    version_added: '1.17.0'
```

Comment:
```suggestion
    version_added: '1.17.0'
```

The duplicate `version_added` is a problem for our sanity checks. Could you please remove one?

Reply:
@mariolenz - I removed the duplicate version_added. Please review